### PR TITLE
add some python codes to imambungo

### PR DESCRIPTION
**Pull request by imambungo**
work in progress.

reference:
- https://docs.google.com/presentation/d/1xWg1Zu-K8P5FYw8J0_6Z3oS98hGoksHINjuH7XG-ydE/edit#slide=id.g6f9d696afd_0_8
- https://docs.google.com/presentation/d/1uVJcvPsMQoxeCii5CGz8b31A3GlakCwFWzKBtrbKezY/edit#slide=id.g73e70ab8a3_0_7

### CODE REVIEW

**Comment by ariefrahmansyah** [comment]
Nice, lanjut OOP ya

**Comment by imambungo** [comment]
Done kk :+1:

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# ignore vim swap files
+*.swp
+
+# ignore python caches
+*.pyc
```
Comment (ariefrahmansyah) [inline .gitignore:5]: thanks for this .gitignore!

**Review by ariefrahmansyah — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (imambungo) [reply .gitignore:5]: :+1:

**Pull request merged**